draft-ietf-httpbis-rand-access-live.xml

```diff
@@ -478,16 +481,23 @@ Content-Range: bytes 1020000-1254567/*
 
 <!-- Possibly a 'Contributors' section ... -->
 
+    <section anchor="RecommendedVLV" title="Recommendations for Very Large Values">
+        <t>
+            While it would be ideal to define a single numerical Very Large Value, there's no single value that would work for all applications and platforms. e.g. JavaScript numbers cannot represent integer values above 2^^53, so a JavaScript application may want to use 2^^53-1 for a Very Large Value. This value, however, would not be sufficient for all applications. So this draft recommends 2^^53-1 (9007199254740991) as a Very Large Value unless an application has a good justification to use a smaller or larger value. e.g. If it's always known that the resource won't exceed a value smaller than the recommended Very Large Value for an application, a smaller value can be used for efficiency. And if it's likely that an application will utilize resources larger than the recommended Very Large Value, a larger value should be used.
```

Comment:
Is this duplicated text?  It's almost the same as the above.  You should probably just forward-reference this section from above.

Given that you have a dedicated section, I would split the paragraph at "So this draft recommends" and drop the "So".

Reply:
Oops - thought I moved the section. Guess I copied during my late-night editing. Fixing...